fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl

```diff
@@ -160,7 +175,7 @@ dunefd_hd_backgrounds_1x2x6:
 
 #temporarily arranging for an alternative background model in HD because of APA geometry not being correct in the gdml
 
-dunefd_backgrounds_common_v3_5:
+dunefd_backgrounds_common_v3_6:
```

Comment:
Just to double check, we are just dropping v3_5?

Reply:
HD v3.5 was prepared and then soon changed to v3.6. So I don't see why we should keep it around in the configs (I am only keeping v3.4 around because we will eventually come back to it, once the APA geometry is fixed)